apps/yarn.lock

```diff
@@ -2804,6 +2816,16 @@ brcast@^3.0.0:
   version "3.0.1"
   resolved "https://registry.yarnpkg.com/brcast/-/brcast-3.0.1.tgz#6256a8349b20de9eed44257a9b24d71493cd48dd"
 
+brfs@^1.4.0:
```

Comment:
Looks like there are a lot of changes in yarn.lock that don't seem directly related to bumping a p5.play version. Do you know where these are coming from?

Reply:
The p5.play version was behind by 4 versions due to a bunch of work done for HOC 2018 that only went into the Dance Party repo at the time. This takes those 4 version updates as well as the version bump for the bugfix.
Most of the dependencies come from this PR: https://github.com/code-dot-org/p5.play/commit/0dd8722785f1700842259cd0174cc369d3b8d457

Reply:
Ahhh, I see. Wow - that's a lot of dependencies!